config/rbac/manager_role.yaml

```diff
@@ -234,6 +234,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - hive.openshift.io
+  resources:
+  - syncsets
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
I can see we have three instances of 
```
- apiGroups:
  - hive.openshift.io
  resources:
  - syncsets
```
IMO we should have a single instance as it will be easier for humans.

Reply:
We decided that a Jira card would be created to decide how to handle roles in the future (manually created / maintained vs autogenerated by kubebuilder).

In the mean time, we will continue with our current pattern of annotating each controller with the specific permissions that it needs, which will generate multiple entries for the same resource in the role.